motor_spec:
    gear_ratio : 213
    wheel_base : 0.140
    wheel_radius : 0.035
    enc_pulse : 44

use_imu_during_odom_calc:
    use_imu : True
    complementary_filter_coef : 0.5